frontend/src/routes/llm-settings.tsx

```diff
@@ -279,13 +279,34 @@ function LlmSettingsScreen() {
               className="flex flex-col gap-6"
             >
               {!isLoading && !isFetching && (
-                <ModelSelector
-                  models={modelsAndProviders}
-                  currentModel={
-                    settings.LLM_MODEL || "anthropic/claude-sonnet-4-20250514"
-                  }
-                  onChange={handleModelIsDirty}
-                />
+                <>
+                  <ModelSelector
+                    models={modelsAndProviders}
+                    currentModel={
+                      settings.LLM_MODEL || "openhands/claude-sonnet-4-20250514"
+                    }
+                    onChange={handleModelIsDirty}
+                  />
+                  {settings.LLM_MODEL?.startsWith("openhands/") && (
+                    <div className="mt-2 text-sm text-blue-400">
+                      <Trans
+                        i18nKey={I18nKey.SETTINGS$OPENHANDS_API_KEY_HELP}
```

Comment:
@OpenHands We should display this HELP when the user hovers on top of "OpenHands" model provider, as well as AFTER the user selects OpenHands as a provider, but not yet saving it to the settings (so we can't check `settings.LLM_MODEL`).

Also, the link you implemented isn't working. We should only add link to the "API Keys Tab" wording, not the whole I18nKey.SETTINGS$OPENHANDS_API_KEY_HELP